### PR TITLE
Adjust code formatter and format code

### PR DESCRIPTION
**Pull request by Beacze**
Checkstyle has below 3k violations

### CODE REVIEW

File: config/Kitodo-IDE-formatting-Eclipse.xml [hunk 1]
```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="12">
```
Comment (henning-gerhardt) [inline config/Kitodo-IDE-formatting-Eclipse.xml]: missing license header

File: config/Kitodo-IDE-formatting.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<code_scheme name="Project">
```
Comment (henning-gerhardt) [inline config/Kitodo-IDE-formatting.xml]: missing license header

File: config/Kitodo-IDE-formatting.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="5" />
+    <option name="JD_ALIGN_PARAM_COMMENTS" value="false" />
+    <option name="JD_ALIGN_EXCEPTION_COMMENTS" value="false" />
+    <codeStyleSettings language="JAVA">
+        <option name="IF_BRACE_FORCE" value="3" />
+        <option name="DOWHILE_BRACE_FORCE" value="3" />
+        <option name="WHILE_BRACE_FORCE" value="3" />
+        <option name="FOR_BRACE_FORCE" value="3" />
+    </codeStyleSettings>
+</code_scheme>
```
Comment (henning-gerhardt) [inline config/Kitodo-IDE-formatting.xml]: missing new line at end of file

File: config/Kitodo-IDE-formatting.xml (hunk 2 above)
Comment (Beacze) [inline config/Kitodo-IDE-formatting.xml]: fixed

File: config/Kitodo-IDE-formatting.xml (hunk 3 above)
Comment (Beacze) [inline config/Kitodo-IDE-formatting.xml]: fixed

File: config/Kitodo-IDE-formatting-Eclipse.xml (hunk 1 above)
Comment (Beacze) [inline config/Kitodo-IDE-formatting-Eclipse.xml]: fixed

**Pull request merged**